services/gossip/codec.go

```diff
@@ -8,7 +8,7 @@ func encodeBlockPair(blockPair *protocol.BlockPairContainer) ([][]byte, error) {
 	if blockPair == nil || blockPair.TransactionsBlock == nil || blockPair.ResultsBlock == nil {
 		return nil, &ErrCodecEncode{"BlockPair", blockPair}
 	}
-	payloads := make([][]byte, 0, 5+
+	payloads := make([][]byte, 0, 5+ // the 5 is coming from the 5 payloads we're always sending first (txHeader, txMetadata...)
```

Comment:
instead of the comment, set 5 to a constant `sizeOfHeaderAndMetadataAndFooAndBarAndBaz`, this is more self explanatory and will make it easier for people to know they need to change it if the number of fixed payloads change

Reply:
Good point